Examples/ImageRegistrationOptimizerWeights/Documentation.rst

```diff
@@ -0,0 +1,39 @@
+Image Registration Optimizer Weights
+====================================
+
+.. include:: registrationExamplePrefix.rst
+
+Overview
+--------
+The image registration framework allows us to disable subsets of the
+transformation parameters so that we can use a lower dimensional
+transformation. Or if we have some better prior knowledge weigh the
```

Comment:
"Of if we have some better prior knowledge *we can* weigh the ..."

Not a complete sentence.

Reply:
Modified.